cpp/pipelines/ixtheo_marc_pipeline_fifo.sh

```diff
@@ -176,17 +176,17 @@ wait
 
 
 StartPhase "Parent-to-Child Linking and Flagging of Subscribable Items"
-make_named_pipe --buffer-size=$FIFO_BUFFER_SIZE GesamtTiteldaten-post-phase"$PHASE"-"${date}".mrc >> "${log}" 2>&1
+#:qmake_named_pipe --buffer-size=$FIFO_BUFFER_SIZE GesamtTiteldaten-post-phase"$PHASE"-"${date}".mrc >> "${log}" 2>&1
 (add_superior_and_alertable_flags ixtheo GesamtTiteldaten-post-phase"$((PHASE-1))"-"${date}".mrc \
                                          GesamtTiteldaten-post-phase"$PHASE"-"${date}".mrc >> "${log}" 2>&1 && \
 EndPhase || Abort) &
 
 
 # Note: It is necessary to run this phase after articles have had their journal's PPN's inserted!
-StartPhase "Populate the Zeder Journal Timeliness Database Table"
-(collect_journal_stats ixtheo GesamtTiteldaten-post-phase"$((PHASE-1))"-"${date}".mrc \
-                              GesamtTiteldaten-post-phase"$PHASE"-"${date}".mrc >> "${log}" 2>&1 && \
-EndPhase || Abort) &
+# StartPhase "Populate the Zeder Journal Timeliness Database Table"
```

Comment:
This phase should be re-enabled again in git (you can disable it locally if necessary for testing, but please dont change it in git)